vue.config.js

```diff
@@ -56,6 +58,11 @@ module.exports = {
         return [definitions];
       })
       .end()
+      .plugin('ssr')
+      .tap((options) => ({
+        entry: (target) => `./src/entry-${target}`,
+        ...options[0],
+      }))
```

Comment:
This part causing an error:
> Error: args must be an array of arguments

Shouldn't it be handled by @akryum/vue-cli-plugin-ssr?

Reply:
Not sure, please check https://vue-cli-plugin-ssr.netlify.app/guide/configuration.html

Reply:
@davidyuk I guess I got what you mean, I didn't define that this option are about `@akryum/vue-cli-plugin-ssr`

Reply:
This is not needed because it is already done on the side of `@akryum/vue-cli-plugin-ssr`

Reply:
thanks, removed